dist/index.html

```diff
@@ -1,24 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <title>Output Management</title>
-    <script defer="defer" src="runtime.bundle.js"></script>
-    <script defer="defer" src="index.bundle.js"></script>
-  </head>
-  <body>
-    <div class="container">
-      <div class="list">
-        <h2>To-do list</h2>
-        <div class="row">
-          <input id="input-box" placeholder="add to your list" />
-          <button onclick="addToList()">Add</button>
-        </div>
-        <ul id="list-container"></ul>
```

Comment:
- @LethaboMot You did a good job working on the functionalities, however, I noticed in your design they are no checkboxes, to make a task complete or not, but this is done through a click on the task. 
    -  I will advise you to add a checkbox to the design and when it is clicked, it should trigger the complete/uncomplete function, as it is stated in the requirement a checkbox is needed, also adding a checkbox will come intuitive to a user on what how to mark a task. (meaning it is unclear how to make a task without the checkbox) , 
- **Your Design**
![image](https://github.com/LethaboMot/weppack2/assets/58771507/ca329849-9b9b-4305-b832-4a3059d84a3d)

- **Intended Design (checkbox)**

![image](https://github.com/LethaboMot/weppack2/assets/58771507/bdc3ab73-20ec-4aa1-9216-7af1462d76d4)